src/data_store.py

```diff
@@ -0,0 +1,72 @@
+from pyspark.sql import SparkSession, DataFrame
+from pyspark.sql.types import *
+from pyspark.sql.functions import *
+from utils import all_columns_to_snake_case
+from app_context import AppContext
+
+class DataStore:
+
+    schema = StructType([
+        StructField("Date", DateType(), False),
+        StructField("Open", DoubleType(), False),
+        StructField("High", DoubleType(), False),
+        StructField("Low", DoubleType(), False),
+        StructField("Close", DoubleType(), False),
+        StructField("Adj Close", DoubleType(), False),
+        StructField("Volume", IntegerType(), False)
+    ])
+
+
+    def __init__(self, app_context: AppContext) -> None:
+        self.spark = app_context.spark
+        self.config = app_context.config
+
+
+    def load_stock_raw(self, symbol: str) -> DataFrame:
+        """ Read CSV, Rename columns """
+        input_dir = self.config["prepare"]["input_dir"]
```

Comment:
Introduce a configuration object for the DataStore
`"data_store": {
         "dir_raw": "data/raw",
         "dir_cln": "data/cln",
         "dir_drv": "data/drv",
}
`
`self.config['data_store']['dir_raw']`